### PR TITLE
Added SourceXpress APIs support and AWG defects fix.

### PR DESCRIPTION
**Pull request by v12ganesh**
## Proposed changes

Please include a summary of the changes and any links to related issues. Please also include relevant motivation and context.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Functionality update (non-breaking change which updates or changes existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] CI/CD update (an update to the CI/CD workflows, scripts, and/or configurations)
- [ ] Documentation update (an update to enhance the user experience when reading through the docs)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have followed the guidelines in the [CONTRIBUTING](https://github.com/tektronix/tm_devices/blob/main/CONTRIBUTING.md) document
- [x] I have signed the CLA
- [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/tektronix/tm_devices/pulls) for the same update/change
- [x] I have performed a self-review of my code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Basic linting passes locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added necessary documentation (if appropriate)
- [x] I have updated the Changelog with a brief description of my changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,13 @@ Valid subsections within a version are:
...
 ## Unreleased
 
-Things to be included in the next release go here.
+### Added
+
+- Support for SourceXpress to AWG70KA, AWG70KB and AWG7K models.
+
+### Fixed
+
+- Fixed APIs with write and query accepting arguments for AWG70KA and AWG70KB models drivers.
```
Comment (nfelt14) [inline CHANGELOG.md:25]:
```suggestion
- Fixed APIs with writes and queries accepting arguments for AWG70KA and AWG70KB models drivers.
```

File: README.md [hunk 2]
```diff
@@ -148,6 +148,7 @@ _Software Solution Support Levels_
 | Software<br>Solution | Command<br>Type | Basic<br>Control | Python API<br>Validation<br>Status |
 | -------------------- | --------------- | ---------------- | ---------------------------------- |
 | DPOJET               | PI              | ✅                | ✅                                  |
+| SourceXpress         | PI              | ✅                | ✅                                  |
```
Comment (nfelt14) [inline README.md:151]: Add an entry to the glossary explaining what this Software Solution is.

**Review by nfelt14 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nfelt14) [reply README.md:151]: You will also need to update the `macros.py` file to properly update the readme when the docs are generated.

**Review by u625355 — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,7 +16,15 @@ Valid subsections within a version are:
 
 ## Unreleased
 
-Things to be included in the next release go here.
+- Things to be included in the next release go here.
```
Comment (nfelt14) [inline CHANGELOG.md:19]:
```suggestion
Things to be included in the next release go here.
```

File: docs/glossary.md [hunk 4]
```diff
@@ -83,6 +83,8 @@ SESR
 SMU
 : Source Measure Unit
 
+SourceXpress
+: An application to Load and play waveforms remotely for Tektronix Arbitrary Waveform Generators
```
Comment (nfelt14) [inline docs/glossary.md:87]:
```suggestion
: An application to load and play waveforms remotely for Tektronix Arbitrary Waveform Generators

```

**Review by nfelt14 — CHANGES_REQUESTED** [review]
You will also need to rebase your branch with the latest from main

File: docs/glossary.md [hunk 5]
```diff
@@ -83,6 +83,8 @@ SESR
 SMU
 : Source Measure Unit
 
+SourceXpress
+: An application to load and play waveforms remotely for Tektronix Arbitrary Waveform Generators
```
Comment (nfelt14) [inline docs/glossary.md:87]: Add whitespace (newline) after this.
```suggestion
: An application to load and play waveforms remotely for Tektronix Arbitrary Waveform Generators

```

**Review by nfelt14 — CHANGES_REQUESTED** [review]

**Comment by nfelt14** [comment]
- [ ] Please sign all of your commits

**Review by nfelt14 — CHANGES_REQUESTED** [review]
Please sign all of your commits

**Review by nfelt14 — APPROVED** [review]

**Review by u625355 — APPROVED** [review]

**Comment by u625355** [comment]
> Please sign all of your commits

Untill you sign all of your commits, PR cannot be merged.

**Comment by v12ganesh** [comment]
Due to merge conflicts and SSH key issue, closing this PR.

**Pull request closed without merging**